osu.Game/Overlays/Wiki/Markdown/WikiMarkdownImage.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using Markdig.Syntax.Inlines;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Containers.Markdown;
+using osu.Framework.Graphics.Cursor;
+using osu.Game.Online.API;
+
+namespace osu.Game.Overlays.Wiki.Markdown
+{
+    public class WikiMarkdownImage : MarkdownImage, IHasTooltip
```

Comment:
This needs some kind of placeholder display (probably with an arbitrary size since we don't know the final size of the image) to show that it's loading. Right now it has a zero size and looks quite weird when it pops in.

Reply:
I'll look into it.

Reply:
I think this may be best left for a follow-up PR.

Reply:
Yeah, sure.